src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1447,6 +1461,8 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
     };
   };
 
+  isPostMerge: boolean;
```

Comment:
For later when we clean up tech debt: does this make sense to define here? This flag is used all throughout the file, so it seems like it would make sense at the top.

Reply:
probably a good idea to move it up in the file, yeah.